### ISSUE TITLE
CCV bayesian score classifications (Ie cold VUS etc) on general report

### ISSUE THREAD

**Issue by dnil** [Clinical-Genomics/scout#5143]
Available for ACMG, should probably be symmetrical.

**Comment by molucorner** [comment]
I could add this, since it's a CCV classification thing and all, but since CCV only uses the point system, I feel like "Bayesian classification" would be confusing. Would "ClinGen-CGC-VICC score" work?

**Comment by dnil** [comment]
Sounds good - one could even cut the "score" or what do you think?

**Comment by molucorner** [comment]
Now that you say it, I guess it's redundant since the temperature indicator starts with "Score" anyways!

Now that I'm comparing the Bayesian classification with the ACMG one, is there any reason the ACMG classification (and by extension, the CCV classification) uses the code rather than the label? Especially for the "likely" classifications it looks nicer without the underscore.

**Comment by dnil** [comment]
Nah, feel free to change both I think? Unless they somehow wrap in an ugly way, but if so we can just fix that instead of having underscores!

**Comment by molucorner** [comment]
Since the two CCVs will always agree on the classification it would probably look cleaner to combine them, maybe like this?

![VUS with scores v2](https://github.com/user-attachments/assets/c1f79694-ba06-406e-aedf-98c1191fa9d7)
![with scores v2](https://github.com/user-attachments/assets/28666cdc-602c-48b2-aca0-ff6e0684b2e0)
